diff --git a/sc2env.py b/sc2env.py
index ab39c19..bb6fc57 100644
--- a/sc2env.py
+++ b/sc2env.py
@@ -85,5 +85,7 @@ class Sc2Env(gym.Env):
 			pickle.dump(data, f)
 
 		# run incredibot-sct.py non-blocking:
-		subprocess.Popen(['python3', 'incredibot-sct.py'])
+		print("Popen now!")
+		subprocess.Popen(['python3', 'incredibot-sct.py'], shell=True)
+		print("Popen gone through")
 		return observation  # reward, done, info can't be included
